Gemfile

```diff
@@ -91,12 +65,14 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "database_cleaner"
   gem "capybara-select2"
-  gem "show_me_the_cookies"
-  gem "brakeman"
+  gem 'brakeman', require: false
   gem "pry"
   gem "pry-remote"
   gem 'pry-byebug'
+  gem 'ruby-prof'
+end
+
+group :development do
```

Comment:
Yeah, we don't currently have custom designs for most error pages, and likely won't until @nnunn is back from mat leave.